samples/bluetooth/peripheral_cgm/README.rst

```diff
@@ -0,0 +1,74 @@
+.. _peripheral_cgm:
+
+Bluetooth: Continuous Glucose Monitoring Service (CGMS)
+#######################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Peripheral Continuous Glucose Monitoring Service (CGMS) sample demonstrates the use of continuous glucose service.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+Overview
+********
+
+The sample demonstrates a basic Bluetooth® Low Energy Peripheral role functionality that exposes the Continuous Glucose Monitoring GATT Service.
+Once it connects to a Central device, it generates dummy glucose measurement values.
+You can use it together with the Continuous Glucose module in the nRF Toolbox app.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_cgm`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, connect to a smartphone
+with the nRF Toolbox installed to test it as follows:
+
+1. |connect_terminal_specific|
+#. Reset the development kit.
+#. Open the nRF Toolbox app and select the **Continuous Glucose** module.
+#. Click the name of the development kit to connect to it.
+   Authentication is required.
```

Comment:
```suggestion
#. Click the name of the development kit to connect to it.

   Authentication is required.
```
Needs an empty line, because the stuff is not actually a part of the step, just additional information between the steps.